/**:
  ros__parameters:
    x_axis: 4
    y_axis: 3
    z_axis: 1
    yaw_axis: 0
    x_vel_max: 1.0
    y_vel_max: 1.0
    z_vel_max: 1.0
    yaw_vel_max: 0.25
